modules/planning/tasks/traffic_decider/crosswalk.cc

```diff
@@ -210,10 +261,16 @@ void Crosswalk::MakeDecisions(Frame* const frame,
   }
 
   for (auto crosswalk_to_stop : crosswalks_to_stop) {
+    // build stop decision
     BuildStopDecision(frame, reference_line_info,
                       const_cast<hdmap::PathOverlap*>(crosswalk_to_stop.first),
                       crosswalk_to_stop.second);
   }
+
+  // update CrosswalkStatus
+  GetPlanningStatus()->clear_crosswalk();
```

Comment:
do you want to reset PlanningStatus::crosswalk()?

Reply:
yes. we'll update crosswalk status. next line with TODO